lib/model/content.dart

```diff
@@ -254,24 +254,40 @@ class QuotationNode extends BlockContentNode {
   }
 }
 
-class CodeBlockNode extends BlockContentNode {
-  // TODO(#191) represent the code-highlighting style spans in CodeBlockNode
-  const CodeBlockNode({super.debugHtmlNode, required this.text});
+class CodeSpanNode extends BlockContentNode {
```

Comment:
```suggestion
class CodeBlockSpan extends InlineContentNode {
```

(Or perhaps `CodeBlockSpanNode`.)

* Just "code span" sounds like a description of what `InlineCodeNode` is. This node is specifically for a span found inside a code block.
* This is a type of inline content node, not a block content node. See the dartdoc comments on InlineContentNode and BlockContentNode.

(Or maybe the base class should even be just `ContentNode`. While it's true that this is an inline content node, it's *not* a node we expect to find in any of the containers that have a generic list of `InlineContentNode`s, i.e. any of the subclasses of `BlockInlineContainerNode` or `InlineContainerNode`. So inheriting directly from `ContentNode` would express that for the benefit of code consuming those container nodes.)

Reply:
Also let's put this after `CodeBlockNode`, rather than before, to match the organization of the other classes in this file.